crates/proto/src/runtime.rs

```diff
@@ -122,6 +122,9 @@ mod tokio_runtime {
     use tokio::task::JoinSet;
     use tokio::time::timeout;
 
+    pub const TLS_TIMEOUT: Duration = Duration::from_secs(5);
+    pub const TCP_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
Nit: keep these at the bottom of the module, please.

Not sure about where we should keep these timeouts. They're arguably not really a related to the runtime, that's merely the lowest level at which we can enforce the TCP timeout (but not the TLS timeout). Maybe we should move these into `src/xfer/mod.rs` or something?

@marcus0x62 thoughts?

Reply:
I think for now, these static timeouts should go in mod.rs as @djc suggests.  In the long-term, I think we'll end up with more timeouts, and will need to make the timeouts configurable; at that point, we'll probably want a struct or something to put these in and pass them around, but that seems like overkill at the moment.